### PR TITLE
OBPIH-3528 Add second page for invoice

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy [hunk 1]
```diff
@@ -134,7 +134,9 @@ class Invoice implements Serializable {
             dateDue: dateDue,
             datePaid: datePaid,
             currencyUom: currencyUom?.id,
-            vendor: party?.id
+            vendor: party?.id,
+            totalCount: invoiceItems.size(),
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy:138]: @DominikaHatala I am getting NPE here. Should be `invoiceItems?.size()?:0`

Comment (DominikaHatala) [reply grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy]: @awalkowiak That's right. Fixed.

File: grails-app/conf/UrlMappings.groovy [hunk 2]
```diff
@@ -255,6 +255,23 @@ class UrlMappings {
...
+            controller = "invoiceApi"
+            action = [GET: "getInvoiceItems"]
+        }
+
+        "/api/invoices/$id/removeItem"(parseRequest: true) {
+            controller = "invoiceApi"
+            action = [DELETE: "removeItem"]
+        }
+
+        "/api/invoices/$id/updateItems"(parseRequest: true) {
```
Comment (jmiranda) [inline grails-app/conf/UrlMappings.groovy:269]: We should be using the request method to tell us what action to perform. The API should just be /api/invoices/:id/items

**Review by jmiranda — APPROVED** [review]
LGTM

**Comment by jmiranda** [comment]
Oops. LGTM except for the one change to the API.

File: grails-app/conf/UrlMappings.groovy (hunk 2 above)
Comment (DominikaHatala) [reply grails-app/conf/UrlMappings.groovy]: Thanks, that's right. I have updated PR.

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**